Acl/SecurityCommentAcl.php

```diff
@@ -45,11 +46,9 @@ class SecurityCommentAcl implements CommentAclInterface
     protected $aclProvider;
 
     /**
-     * The current Security Context.
-     *
-     * @var SecurityContextInterface
+     * @var AuthorizationCheckerInterface|SecurityContextInterface
      */
-    protected $securityContext;
+    protected $authorizationChecker;
```

Comment:
renaming a protected property is a BC break. So it is not good for patch releases